src/cards/ha-media_player-card.html

```diff
@@ -248,8 +248,20 @@
 
   playerObjChanged: function (playerObj) {
```

Comment:
There is a second argument to this function that could help with avoiding trying to catch the picture twice: `oldPlayerObj`. In that case you can avoid doing any image calculation/attr setting if the picture url is the same.